plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -751,13 +806,25 @@ func benchmarkExpire(b *testing.B, podNum int) {
 	}
 }
 
-func makeBasePod(nodeName, objName, cpu, mem string, ports []v1.ContainerPort) *v1.Pod {
+type testingMode interface {
+	Fatalf(format string, args ...interface{})
+}
+
+func makeBasePod(t testingMode, nodeName, objName, cpu, mem, oir string, ports []v1.ContainerPort) *v1.Pod {
 	req := v1.ResourceList{}
 	if cpu != "" {
 		req = v1.ResourceList{
 			v1.ResourceCPU:    resource.MustParse(cpu),
 			v1.ResourceMemory: resource.MustParse(mem),
 		}
+		if oir != "" {
+			if len(strings.Split(oir, ":")) != 2 {
+				t.Fatalf("Invalid OIR string: %v", oir)
```

Comment:
@k82cn Ahh, I agree with your point. I was not even checking `len(strings.Split(oir, ":")` in initial commit on the same assumption that oir format will always be right because its hard coded and additional checks here are like testing the test case.
But then @gyliu513 and @xiangpengzhao insisted that it would be better to have len check. Just for their sake, I added this len check and then extended to this `t` check :)

@gyliu513 @xiangpengzhao , Can I go ahead with the assumption that OIR format will always be correct and not have len check?